### PR TITLE
Adds new methods for inventory types

### PR DESCRIPTION
**Pull request by gzimh**
References CAR-6025

## Motivation and context
Adds fetch methods for:
- `inventory/first-publishing-date-age-groups`
- `inventory/last-price-modified-date-age-groups`

### CODE REVIEW

**Review by masone — APPROVED** [review]
please document the additional functions in the README before merging. don't forget to apply a semantic commit message

File: README.md [hunk 1]
```diff
@@ -108,6 +108,10 @@ Also accompanying modes and param types, as well as default values, are exported
   - [`fetchOptions`](https://carforyou-service.preprod.carforyou.ch/swagger-ui/index.html#/Inventory%20Data/getOptionsUsingGET)
   - [`fetchSeats`](https://carforyou-service.preprod.carforyou.ch/swagger-ui/index.html#/Inventory%20Data/findDistinctSeatsUsingGET)
   - [`fetchTransmissionTypes`](https://carforyou-service.preprod.carforyou.ch/swagger-ui/index.html#/Inventory%20Data/getTransmissionTypesUsingGET)
+  - [`fetchImagesCountGroups`](https://carforyou-service.preprod.carforyou.ch/swagger-ui/index.html#/Inventory%20Data/getImagesCountGroupsUsingGET)
```
Comment (gzimh) [inline README.md:111]: @masone I also documented this one since I saw it was missing from the README 👍

**Comment by disufi** [comment]
@masone shall we reword the commits https://github.com/carforyou/carforyou-api-client-pkg/pull/238/commits ? Or is it ok this way?

**Review by masone — APPROVED** [review]

**Comment by disufi** [comment]
@masone 🤦 Now I get is, your comment 

> don't forget to apply a semantic commit message

was meant for the squash & merge, right? :)

**Comment by masone** [comment]
yeah, it depends on how you choose to merge. single commits are fine if they have semantic messages individually that make sense as atomic changes in the resulting changelog. most people squash-merge, leaving you with the need to giving a semantic commit message then. as it takes the PR title by default, that's sometimes overlooked

**Pull request merged**